components/dropdown/style/index.less

```diff
@@ -49,7 +49,7 @@
     background-clip: padding-box;
 
     &-item,
-    &-submenu-title {
+    &-submenu {
```

Comment:
If I am wrong, maybe you can refuse the PR, but I still want to know what's the correct way to fix the problem.